spec.bs

```diff
@@ -633,12 +633,16 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
         1. Let |biddingScript| be the result of [=fetching resource=] with |ig|'s
           [=interest group/bidding url=], and "text/javascript". If |biddingScript| is an error,
           [=iteration/continue=].
-        1. TODO: Let |generatedBid| be the result of [=evaluating a bidding script=] with
+        1. TODO: Let |generatedBidResult| be the result of [=evaluating a bidding script=] with
            |biddingScript| and « |interestGroup|, |auctionSignals|, |perBuyerSignals|,
            |trustedBiddingSignals|, |browserSignals| ».
-        1. If |generatedBid|'s [=generated bid/bid=] is negative or 0, [=iteration/continue=].
+        1. If |generatedBidResult| is an [=abrupt completion=], [=iteration/continue=].
+        1. Let |generatedBidIdl| be the result of [=converted to an IDL value|converting=] |generatedBidResult| to
+          a {{GenerateBidOutput}}.
+        1. If <a spec="webidl">an exception was thrown</a> in the previous step, [=iteration/continue=].

```

Comment:
See https://github.com/whatwg/webidl/pull/1293#issuecomment-1507787615 for how to do this instead w/ "catch"

Reply:
There's no <dfn> for catch yet [0] -- should it just something like this?

If any exception was caught, [=iteration/continue=].

[0] https://github.com/whatwg/webidl/pull/1293#issuecomment-1508961934

Reply:
Yeah, we can just add it via prose for now and add a TODO to link it if gets exported.